tests/input/src/main/scala/blacklist/NoNullNoCast

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
The filenames don't have suffixes because these files don't need to be compiled (and can't be compiled at the same time because of different dialects they use). Suffixes will be added where a tasty file is needed.

Reply:
I think it would be good to add the `.scala` suffix because editors would enable the Scala syntax highlighting. If we don’t want to compile these files, we should remove the definition of the project `testsInput` in the build definition (so that these files are not anymore part of “Scala source files”).

Reply:
Fixed in [3c44497](https://github.com/civp/language-features-a-la-carte/pull/4/commits/3c444974b90a7b484248e84ace1289c6bb15ad3e). But I will keep the definition of the project `testsInput` because we need to compile source files for semantic checkers to have access to tasty files.